src/workloads/query/TelemetryQueryShapes.yml

```diff
@@ -0,0 +1,133 @@
+      maxPoolSize: 400
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Repeat: &Repeat 1000
+  Threads: &Threads 32
+  MaxPhases: &MaxPhases 4
+  nop: &Nop {Nop: true}
+  dbname: &db test
+
+ActorTemplates:
+- TemplateName: ComputeFindQuery
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "ComputeFindQuery"}}
+    Type: CrudActor
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: {unused: "please specify in which phases this actor should be active."}}}]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: *Repeat
+          Database: *Database
+          Collection: *Collection
+          Operations:
+          - OperationName: find
+            OperationCommand:
+              Filter: {^Parameter: {Name: "Filter", Default: {a: 1, b: 2, x0: {$exists: false}}}}
+
+Actors:
+# Clear any pre-existing collection state.
+- Name: ClearCollection
+  Type: CrudActor
+  Database: *Database
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: drop
+
+- Name: InsertData
+  Type: Loader
+  Threads: 4
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          a1: &integer {^RandomInt: {min: -100, max: 100}}
+          a2: *integer
+        Indexes:
+        - keys: {a1: 1, a2: 1}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+
+# 32 threads with distinct query shapes.
+- ActorFromTemplate:
+    TemplateName: ComputeFindQuery
+    TemplateParameters:
+      Name: "AlternatingShapes"
+      Threads: 32
+      Filter:
+        # Will find around 0.005 * 0.005 * 1e6 = 2.5 documents.
+        {$and:
+           [{a1: 1}, {a2: 1}, {^Object: {
+             withNEntries: 1,
+             havingKeys: {^RandomString: {length: 10}},
+             andValues: {$exists: false},
+             duplicatedKeys: skip
+           }}]
+        }
+      OnlyActiveInPhase: 3
+
+# 32 threads with the same query shape.
+- ActorFromTemplate:
+    TemplateName: ComputeFindQuery
+    TemplateParameters:
+      Name: "OneShape"
+      Threads: 32
+      Filter: 
+        # Will find around 0.005 * 0.005 * 1e6 = 2.5 documents.
+        {$and:
+           [{a1: 1}, {a2: 1}, {^Object: {
+             withNEntries: 1,
+             havingKeys: "x0",
```

Comment:
Can you add a comment here about how we could simplify this but want to leave it the same just to rule out the possibility of a perf impact there?